images/nginx/build.sh

```diff
@@ -204,6 +204,14 @@ export MAKEFLAGS=-j${CORES}
 export CTEST_BUILD_FLAGS=${MAKEFLAGS}
 export HUNTER_JOBS_NUMBER=${CORES}
 
+# Installing luarocks packages
+if [[ ${ARCH} == "x86_64" ]]; then
+  luarocks install lrexlib-pcre 2.7.2-1
+fi
+
+luarocks install lua-cjson
```

Comment:
We already have cjson being installed at https://github.com/kubernetes/ingress-nginx/pull/2588/files#diff-bd50bd51e758b65562491458b5a6f46dR88.

Reply:
It's ok if I remove that and we use luarocks? For some reason that package is not being installed in arm

Reply:
oh interesting, yeah we can use luarocks instead, but could you fix version?

Reply:
done